rabbitmqclient/rabbitmq_client_factory.go

```diff
@@ -81,12 +81,12 @@ func generateRabbitholeClient(connectionCreds map[string]string, tlsEnabled bool
 		cfg.RootCAs = certPool
 
 		transport := &http.Transport{TLSClientConfig: cfg}
-		rabbitmqClient, err = rabbithole.NewTLSClient(fmt.Sprintf("%s", string(uri)), string(defaultUser), string(defaultUserPass), transport)
+		rabbitmqClient, err = rabbithole.NewTLSClient(fmt.Sprintf("%s", uri), defaultUser, defaultUserPass, transport)
 		if err != nil {
 			return nil, fmt.Errorf("failed to instantiate rabbit rabbitmqClient: %v", err)
 		}
 	} else {
-		rabbitmqClient, err = rabbithole.NewClient(fmt.Sprintf("%s", string(uri)), string(defaultUser), string(defaultUserPass))
+		rabbitmqClient, err = rabbithole.NewClient(fmt.Sprintf("%s", uri), defaultUser, defaultUserPass)
```

Comment:
`connectionCreds` is a map of string to string, and we get `uri` from this map. Can we just pass `uri` and remove the `fmt.Sprintf()` call?

Reply:
Sure! Great eye for spotting that. Updated